browser/CapacitorBrowser.podspec

```diff
@@ -0,0 +1,18 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name = 'CapacitorBrowser'
+  s.version = package['version']
+  s.summary = package['description']
+  s.license = package['license']
+  s.homepage = package['repository']['url']
+  s.author = package['author']
+  s.source = { :git => package['repository']['url'], :tag => s.version.to_s }
+  s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
+  s.ios.deployment_target  = '11.0'
+  s.dependency 'Capacitor'
+  s.framework = 'SafariServices'
```

Comment:
Is this needed? We don't have it in the Capacitor.podspec and the plugin works. 

In Objective-C, since iOS 7 you can use `@import SafariServices` instead of `#import <SafariServices/SafariServices.h>` and that autolinks the SafariServices framework. Looks like the Swift import works the same way.

Reply:
I don't know? I didn't notice that capacitor omitted it. AFAIK, any of the system frameworks are automatically linked by Xcode (unless you want them to be optionally linked); so it should build without it. But it seems like that dependency should still be documented?